src/coreclr/tests/runtest.cmd

```diff
@@ -331,7 +331,7 @@ REM Skip mscorlib since it is already precompiled.
 if /I "%3" == "mscorlib.dll" exit /b 0
 if /I "%3" == "mscorlib.ni.dll" exit /b 0
 
-"%1\crossgen.exe" /Platform_Assemblies_Paths "%CORE_ROOT%" "%2" >nul 2>nul
+"%1\crossgen.exe" /nologo /Platform_Assemblies_Paths "%CORE_ROOT%" "%2" >nul 2>nul
```

Comment:
I did not test this bonus one.

Reply:
This one seems to be unnecessary as we redirect stdout and stderr to nul here.

Reply:
This is true. Still, I'll leave it for consistency and in case that changes.